rasa_core/policies/fallback.py

```diff
@@ -92,8 +92,12 @@ def predict_action_probabilities(self, tracker, domain):
         # it is set to 1.0 here in order
         # to not override standard behaviour
         nlu_confidence = nlu_data["intent"].get("confidence", 1.0)
-
-        if tracker.latest_action_name == self.fallback_action_name:
+        if force:
+            logger.debug("Action listen was predicted after a user message. "
+                         "Predicting fallback action: {}"
+                         "".format(self.fallback_action_name))
+            score = 1.1
```

Comment:
lets move the `result` creation into a separate function

```python
def fallback_scores(self, domain, fallback_score):
    """Prediction scores used if a fallback is necessary."""
    
    result = [0.0] * domain.num_actions
    idx = domain.index_for_action(self.fallback_action_name)
    result[idx] = fallback_score
    return result
```
and then we can use that function in all the existing if branches here and instead of using the `force` parameter we do not call `predict_action_probabilities` from the policy ensemble in the action listen fallback case, but instead we use the above `fallback_scores`. Feel free to also make the `1.1` a constant so we can use it from the ensemble.